boards/arm/qemu_cortex_m0/nrf_timer_timer.c

```diff
@@ -15,10 +15,12 @@
 
 #define TIMER NRF_TIMER0
 
-#define COUNTER_MAX 0xffffffff
+#define COUNTER_MAX (0xFFFFFFFF)
```

Comment:
I'm new to Zephyr, but is there a style guide reason to put constants inside brackets?  If not, they add noise so please remove them.

Reply:
Removed now @nzmichaelh , thanks.